### PR TITLE
Hubspot: v2.0

### PR DESCRIPTION
**Pull request by erinkcochran87**
This PR updates the HubSpot documentation for version 2.x.

**Release notes/version summary**: 

- Foreign key references included in schema (`_integration-schemas/hubspot/foreign-keys.md`) for version 2.x only.
- The `contacts` table now uses the `versionTimestamp` field as a Replication Key (bookmark) (singer-io/tap-hubspot#58)
- Form submissions in the `contacts` table now includes **all** submissions, not just the newest (singer-io/tap-hubspot#61)
- `contacts_by_company` can now be selected in the UI for replication (singer-io/tap-hubspot#63)
- Field selection is now supported (singer-io/tap-hubspot#67)
- The `keywords` table/stream was removed due to HubSpot deprecation (singer-io/tap-hubspot#68)
- Additional `counter` event fields added to the `campaigns` table schema (singer-io/tap-hubspot#69)
- `isDeleted` fields were removed from `companies` and `deals` (singer-io/tap-hubspot#70)

### CODE REVIEW

**Comment by erinkcochran87** [comment]
@KAllan357 [Here's the rendered preview](https://deploy-preview-7--stitchdocs.netlify.com/docs/integrations/saas/hubspot/) of the new docs. Take and let me know if anything can be improved :) Thanks!

File: _integration-schemas/hubspot/companies.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+version: "2.0"
 
 name: "companies"
 doc-link: http://developers.hubspot.com/docs/methods/companies/get_company
 singer-schema: https://github.com/singer-io/tap-hubspot/blob/master/tap_hubspot/schemas/companies.json
 description: |
   The `companies` table contains info about the companies your HubSpot contacts belong to.
 
-  **Note:** when this table is synced, the [`hubspot_contacts_by_company`](#hubspot_contacts_by_company) table will also be automatically synced and created in your data warehouse.
+  **Note:** When this table is set to replicate, the [`contacts_by_company`](#contacts_by_company) table will be automatically created in your data warehouse.
```
Comment (KAllan357) [inline _integration-schemas/hubspot/companies.md]: This is actually not true. We have table selection here so you can choose whether or not to sync this table.

**Comment by KAllan357** [comment]
Left a comment for one thing, but otherwise :+1: :100: :rocket: :racehorse:

File: _integration-schemas/hubspot/companies.md (hunk 1 above)
Comment (erinkcochran87) [reply _integration-schemas/hubspot/companies.md]: Oooh, this is great! Before this wasn't selectable in the UI. I'll make this change in just a sec, but then these docs are good to go.

**Pull request merged**